.github/workflows/wheels.yml

```diff
@@ -0,0 +1,49 @@
+name: Wheels
+
+on: [push, pull_request]
+
+env:
+  REPO_DIR: .
+  BUILD_DEPENDS: ""
+  TEST_DEPENDS: "pytest pytest-cov"
+  WHEEL_SDIR: wheelhouse
```

Comment:
I think these environment variables are for a multibuild setup, not for cibuildwheel, you should try see what are the equivalent ones that cibuildwheel reads